concordia/static/scss/action-app.scss

```diff
@@ -3,12 +3,18 @@
 @import '../../../node_modules/bootstrap/scss/mixins';
 
 $concordia-app-footer-height: $line-height-lg * 1rem;
+$font-size-xs: $font-size-base * .8125; // Move to global variables location
```

Comment:
If you wanted to create something like `concordia/static/scss/_variables.scss` or `concordia/static/scss/_common.scss` I'd support doing that now. It seems like a good convention to establish.

Reply:
Agree, it's been added.